domains/client/block-preprocessor/src/lib.rs

```diff
@@ -103,7 +103,7 @@ where
 
     let shuffling_seed = consensus_client
         .runtime_api()
-        .extrinsics_shuffling_seed(block_hash, header)?;
+        .extrinsics_shuffling_seed(header.hash())?;
```

Comment:
Why not using `block_hash` anymore? `header.hash()` will unnecessarily compute the hash and is more expensive if you already have the hash. Or was this a fix for some issue?

Reply:
no, it can be block_hash as well 👍🏼